susemanager/src/mgr_bootstrap_data.py

```diff
@@ -2077,6 +2081,12 @@
         "DEST": DOCUMENT_ROOT + "/pub/repositories/raspbian/12/bootstrap/",
         "TYPE": "deb",
     },
+    "fuss-12-amd64-uyuni": {
+        "BASECHANNEL": "fuss-12-pool-amd64-uyuni",
+        "PKGLIST": PKGLISTFUSS12,
```

Comment:
As said above.

Reply:
`PKGLISTFUSS12` was fixed, the variable is right here. Same as for Raspberry Pi OS 12.